CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4.0
+
+- Added support for add-to-app Flutter modules. Use `PfsTarget=` to set Fullstory version on Android if you encounter issues.
```

Comment:
Maybe add a link here to clarify what we're talking about?

```suggestion
- Added support for [add-to-app](https://github.com/flutter/samples/tree/main/add_to_app/prebuilt_module) Flutter modules. Use `PfsTarget=` to set Fullstory version on Android if you encounter issues.
```

(Or perhaps it should link to https://github.com/flutter/samples/tree/main/add_to_app - which do you think makes more sense?)

Reply:
oh, and `PfsTarget=` should have a dash. I'll update my suggestion above.

Reply:
Good idea. I used the actual docs page that links to the repo. I figure more context is better?